plugins/MauticCrmBundle/Integration/ConnectwiseIntegration.php

```diff
@@ -127,6 +127,14 @@ public function getApiUrl()
         return sprintf('%s/v4_6_release/apis/3.0', $this->keys['site']);
     }
 
+    /**
+     * @return \MauticPlugin\MauticCrmBundle\Api\ConnectwiseApi
+     */
+    public function getApiHelper()
+    {
+        return parent::getApiHelper();
```

Comment:
What's the reason for overriding this method?

Reply:
Probably for a IDE typehinting but you can use `@method` in the class doc block to provide this instead of overriding the method